solr/conf/schema.xml

```diff
@@ -329,11 +329,17 @@
 
   </fields>
 
- <!-- Field to use to determine and enforce document uniqueness.
+ <!-- Field to use to determine and enforce document uniqueness. 
       Unless this field is marked with required="false", it will be a required field
    -->
  <uniqueKey>id</uniqueKey>
 
+ <!-- field for the QueryParser to use when an explicit fieldname is absent -->
+ <!--  <defaultSearchField>text</defaultSearchField> -->
+
+ <!-- SolrQueryParser configuration: defaultOperator="AND|OR" -->
+ <solrQueryParser defaultOperator="OR"/>
```

Comment:
Specifically what happened here?

Reply:
these are the live configs I merged from prod when I was trying to figure out which set of configs was actually being used since there's both a `solr/conf` and a `solr/config`. So this gets us closer to how SOLR should be running, in theory